spec/login_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```

Comment:
I'm glad to see acceptance testing in the early stage
Good job! 😄 

I'd just put this specs into a `features` or `system` directory. It's a common practice when using Capybara & RSpec 

https://github.com/teamcapybara/capybara#using-capybara-with-rspec

Reply:
Thanks, we read the information from the link and changed the file to spec / features